src/components/Chat/ChatContent/ChatContent.tsx

```diff
@@ -95,7 +95,7 @@ const ChatContent = () => {
               <DownloadChat saveRef={saveRef} />
             )}
           </div>
-          <div className='w-full h-36'></div>
+          <div class="w-full h-36" style="height:2rem;"></div>
```

Comment:
This project uses [tailwind css](https://tailwindcss.com/). Use of `style` can be omitted. In this case you can just change the `h-36` class as you wanted.